### PR TITLE
Send JS errors to sentry

### PR DESCRIPTION
**Pull request by shanear**
connects #1063 

requires this to be turned on in prod:
https://github.com/department-of-veterans-affairs/appeals-deployment/pull/540

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -4,6 +4,14 @@
   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
 
   <title>Caseflow <%= logo_name %></title>
+
+  <% if ENV["PUBLIC_SENTRY_DSN"] %>
+    <script src="https://cdn.ravenjs.com/3.6.0/raven.min.js"></script>
```
Comment (shanear) [inline app/views/layouts/application.html.erb]: Grabbing this from the public CDN right now cause it's easy. We can move it into our CDN after the fact if we think that's safer.

Comment (askldjd) [inline app/views/layouts/application.html.erb]: Hmm... I am okay with this outside of VA network, but NSOC security team _will likely_ flag this as a XSS vulnerability.

Comment (shanear) [inline app/views/layouts/application.html.erb]: @askldjd rails sanitizes that by default

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -4,6 +4,14 @@
...
+    <script>
+      Raven.config("<%= ENV["PUBLIC_SENTRY_DSN"] %>").install();
```
Comment (askldjd) [inline app/views/layouts/application.html.erb]: Where is PUBLIC_SENTRY_DSN defined?

Comment (askldjd) [inline app/views/layouts/application.html.erb]: NM, just saw the other PR...

**Pull request merged**